base/typography.scss

```diff
@@ -57,6 +62,15 @@ p {
   margin: 0 0 ($base-spacing / 2);
 }
 
+button {
```

Comment:
![no.](https://dl.dropboxusercontent.com/u/9061771/AD%20GIF/Michael%20Bluth%20No%20No%20No.gif)

There is a whole button's file! 

* Don't extend `h3` like that. All you're inheriting is the font-size and text-transform, and those aren't something that should be universal for buttons.

`min-width` of `300px` will break small submit buttons. If you just want this for forms I would scope it to that only.